tools/appveyor.psm1

```diff
@@ -342,6 +348,32 @@ function Invoke-AppVeyorTest
     Set-BuildVariable -Name TestPassed -Value True
 }
 
+function Compress-CoverageArtifacts
+{
+    # Create archive for test content, OpenCover module and CodeCoverage build
+    if(Test-DailyBuild)
```

Comment:
Removed the check.

---
In reply to: [89253173](https://github.com/PowerShell/PowerShell/pull/2760#discussion_r89253173) [](ancestors = 89253173)